megamek/src/megamek/common/net/marshall/NativeSerializationMarshaller.java

```diff
@@ -48,7 +50,7 @@ public void marshall(Packet packet, OutputStream stream) throws Exception {
      */
     @Override
     public Packet unmarshall(InputStream stream) throws Exception {
-        ObjectInputStream in = new ObjectInputStream(stream);
+        ObjectInputStream in = new SerialKiller(stream, Objects.requireNonNull(getClass().getClassLoader().getResource("./megamek/serialkiller.xml")).toString());
```

Comment:
I just noticed the game hangs when I launch using the unixDistTar package, but not when I run with gradlew. Do I need to move the shipped serialkiller.xml file to the mmconf folder instead of the resources folder to use it here?

Reply:
This should be fixed with 3cc0cbd